packages/test-helper-file/src/file.js

```diff
@@ -11,6 +11,23 @@ var Magic = require('mmmagic');
 
 var File = module.exports = {
   fetch: function fetch(filename) {
+    return File.fetchWithoutMagic(filename)
+      .then(function(data) {
+        return new Promise((resolve, reject) => {
+          var magic = new Magic.Magic(Magic.MAGIC_MIME_TYPE);
+          magic.detect(data, function(err2, res) {
+            if (err2) {
+              reject(err2);
+              return;
+            }
+            data.type = res;
+            resolve(data);
+          });
+        });
+      });
+  },
+
+  fetchWithoutMagic: function fetchWithoutMagic(filename) {
```

Comment:
yea. it started out tongue-in-cheek, but fetchWithoutType doesn't work because in a browser, type will actually be determined by the browser.

The only other improvement I can think of is `fetchWithoutMmmagic` and that just seems challenging.